### ISSUE TITLE
USER ISSUE: Crash to Menu upon repairing shovel

### ISSUE THREAD

**Issue by Summerotter** [StrangeLoopGames/EcoIssues#7444]
**Version:** 0.7.3.3 beta

**Steps to Reproduce:**
1. Creative mode
2. Broken Shovel from /give shovel
3. Stone in inventory
4. repair station

**Expected behavior:**
Repaired the shovel

**Actual behavior:**
The following error message popped up:
```
Caught exception

<size=60.00%>Object reference not set to an instance of an object.
   at Eco.Gameplay.Items.Inventory.TryRemoveItems(ItemStack stack, User user)
   at Eco.Gameplay.Components.RepairComponent.Repair(Player player)

Exception has been thrown by the target of an invocation.
   at Eco.Shared.Networking.RPCManager.TryInvoke(Object controller, String methodname, BSONObject bsonArgs, Object& result)
   at Eco.Shared.Networking.RPCManager.InvokeOn(BSONObject bson, Object controller, String methodname)
   at Eco.Plugins.Networking.Clients.Client.ViewRPC(Int32 id, String methodname, BSONObject bson)</size>
```

**Comment by craig-jong** [comment]
Was the item a default shovel? I can fix this, but the base items are not intended for in-game use.

**Comment by Summerotter** [comment]
Yeah it was the default shovel. I'm willing to bet it was the lack of a repair cost that did this.

**Comment by craig-jong** [comment]
Will be fixed in the next 7.4 staging build.
